src/app/components/RankingBox.js

```diff
@@ -0,0 +1,86 @@
+export class RankingBox {
+    constructor(id, scores) {
+      this.render(id, scores);
+    }
+
+      
+    renderContent(id, scores) {
+        const rankingBoxContainer = document.querySelector(`#${id}`);
+
+        this.renderHeader(rankingBoxContainer);
+
+        this.renderTable(scores, rankingBoxContainer);
+    }
+
+    render(id, scores) {
+        this.renderContent(id, scores);
+    }
+
+    numberToOrdinal(number) { 
```

Comment:
W tym przypadku zostawmy switcha bo mamy malo rozbudowana akcje, ale przy większych/ bardziej skomplikowanych pamiętaj że switch jest kiepski pod względem wydajności i jakości kodu dlatego zazwyczaj szukamy jakis alternatyw.

Reply:
dzięki! jeśli chodzi o alternatywy dla switcha to może być if? jak np. poniżej czy jeszcze jakieś inne?

 numberToOrdinal(number){
        if (number === 1){
            return "1st";
        } else if (number === 2) {
            return "2nd";
        } else if (number === 3) {
            return "3rd";
        }
    }